extensions/windows-event-log/ConsumeWindowsEventLog.h

```diff
@@ -101,14 +100,12 @@ class ConsumeWindowsEventLog : public core::Processor
   
 
 protected:
-  bool subscribe(const std::shared_ptr<core::ProcessContext> &context);
-  void unsubscribe();
-  int processQueue(const std::shared_ptr<core::ProcessSession> &session);
+  void processEventsAfterBookmark(core::ProcessSession& session);
+  void processEventRender(const EventRender& renderedData, core::ProcessSession& session);
   wel::WindowsEventLogHandler getEventLogHandler(const std::string & name);
   bool insertHeaderName(wel::METADATA_NAMES &header, const std::string &key, const std::string &value);
   void LogWindowsError();
-  void processEvent(EVT_HANDLE eventHandle);
-  bool processEventsAfterBookmark(EVT_HANDLE hEventResults, const std::wstring& channel, const std::wstring& query);
+  bool processEvent(EVT_HANDLE eventHandle, EventRender& renderedData);
```

Comment:
This member function deserves a Doxygen-style comment describing the behavior and the meaning of the return value, since it's not obvious by looking at the signature.

Reply:
Fixed.